### PR TITLE
Removed unnecessary view, added JTextArea to display info in frame, m…

### PR DESCRIPTION
**Pull request by MC-Student**
…issing correct dagger injections for the JTextArea

### CODE REVIEW

**Comment by MC-Student** [comment]
Test on service passes.

**Comment by andrewoid** [comment]
But does the code run now?

**Comment by MC-Student** [comment]
No, and I know it's because it is missing the correct dagger injections/tags for the JTextArea (as noted in the PR commit message), but I'm unsure how to proceed with that.

**Comment by andrewoid** [comment]
> No, and I know it's because it is missing the correct dagger injections/tags for the JTextArea (as noted in the PR commit message), but I'm unsure how to proceed with that.

How did we solve this in Weather?

File: src/test/java/cohen/mvc/FreebiePostsControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        FreebiePostsController controller = new FreebiePostsController(service, postList);
+        PostListInfo postListInfo = mock();
+        Observable<PostListInfo> observableList = Observable.just(postListInfo);
+        doReturn(observableList).when(service).getPostList();
+
+        //when
+        controller.refreshPosts();
+
+        //then
+        verify(service).getPostList();
```
Comment (andrewoid) [inline src/test/java/cohen/mvc/FreebiePostsControllerTest.java:38]: You need to test that what happens in the subscribe() part happens. 
It's not enough to just make sure getPostList() is called.

**Pull request merged**